app/models/variable.rb

```diff
@@ -10,6 +10,9 @@ class Variable < ApplicationRecord
   belongs_to :value_type
   belongs_to :entity
 
```

Comment:
@Br3nda I'm interested in your opinion here about has many through vs has and belongs to many